cross_valid_total.py

```diff
@@ -0,0 +1,197 @@
+
```

Comment:
What is this file for? There seems to be a lot of duplicate code from `cross_validate_mpi.py`.